web/src/views/repo/pipeline/PipelineWrapper.vue

```diff
@@ -31,6 +31,17 @@
             :is-loading="isRestartingPipeline"
             @click="restartPipeline"
           />
+          <Button
+            v-if="pipeline.status === 'success'"
+            class="flex-shrink-0"
+            :text="$t('repo.pipeline.actions.deploy')"
+            @click="showDeployPipelinePopup = true"
+          />
```

Comment:
Not sure if this is the nicest place for the long term, but I am fine with it for now.